bricks/ai/src/chat-view/hooks/useChatViewInfo.ts

```
@@ -390,80 +434,83 @@
+          type: "markdown",
+          // 只有普通文本类型才更新聊天文本内容，其他如 `tool_response` 不更新。
+          text:
+            msgItemData.type == null || msgItemData.type === "TEXT"
+              ? `${isUpdate ? chattingMessageItem.current!.content.text : ""}${msgItemData.delta.content}`
+              : (chattingMessageItem.current?.content.text ?? ""),
+        },
+        chatting: true,
+        toolCalls:
+          msgItemData.type === "tool_call"
+            ? [
+                ...(chattingMessageItem.current?.toolCalls ?? []),
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)